debezium-connector-oracle/src/test/java/io/debezium/connector/oracle/StreamingDatatypesIT.java

```diff
@@ -67,6 +68,12 @@ protected Builder connectorConfig() {
 
     @Override
     protected boolean insertRecordsDuringTest() {
+        try {
+            connection.executeWithoutCommitting("ALTER SESSION SET TIME_ZONE = '00:00'");
+        }
+        catch (SQLException e) {
+            // ignore
+        }
```

Comment:
Should we consider changing the method signature to allow `SQLException` to be thrown or at the very least wrap it as a `RuntimeException`?  I don't think it makes sense to ignore it here.